src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2407,6 +2397,10 @@ codet java_bytecode_convert_methodt::convert_instructions(
       results[1]=op[0];
       results[0]=op[1];
     }
+    else if(statement=="nop")
```

Comment:
Perhaps this could go into a separate pull request to master rather than test-gen-support? For this unit test we could just use some other dummy instructions (e.g. `iconst_0`). @mgudemann